commands/evalsha_ro.md

```diff
@@ -1,4 +1,6 @@
-This is a read-only variant of the `EVALSHA` command that isn't allowed to execute commands that modify data.
+This is a read-only variant of the `EVALSHA` command that cannot execute commands that modify data.
 
- Unlike `EVALSHA`, scripts executed with this command can always be killed and never affect the replication stream.
- Because it can only read data, this command can always be executed on a master or a replica.
+Unlike `EVALSHA`, scripts executed with this command can always be killed and never affect the replication stream.
+Because it can only read data, this command can always be executed on a master or a replica.
+
+For more information about `EVALSHA` scripts please refer to [Introduction to Eval Scripts](/topics/evalintro)
```

Comment:
Missing period in the end here. This sentence is copy-pasted in many files. (I commented on some and they're not marked as outdated.)